client/my-sites/hosting/main.js

```diff
@@ -35,14 +37,32 @@ const DataSection = ( { title, data } ) => {
 	);
 };
 
-const Hosting = ( { translate } ) => {
+const Hosting = ( { translate, siteId } ) => {
+	const [ loadingPMA, setLoadingPMA ] = useState( false );
 	const dummyInfo = {
 		[ translate( 'URL' ) ]: 'sftp1.wordpress.com',
 		[ translate( 'Port' ) ]: 22,
 		[ translate( 'Username' ) ]: 'test',
 		[ translate( 'Password' ) ]: 'test',
 	};
 
+	function handlePMALogin() {
+		setLoadingPMA( true );
+
+		// If we don't want the pma url to pass through the redux store we can just do a raw wpcom.req here
+		// TODO: If we proceed with this it needs to be a POST, but setting apiNamespace is not currently working for POST
+		wpcom.req
+			.get( `/sites/${ siteId }/hosting/pma`, {
```

Comment:
If folks decide to go with an API here, we can update this to at least dispatch an HTTP action and use the data-layer. (Try searching for usages of `apiNamespace` under the `client/state/data-layer` folder.

Reply:
yeh, it is weird, there are lots of examples of wpcom/v2 being used for a post, but doesn't work when I try it either in requestHttpData or wpcom.req.post - will be something simple I am overlooking for sure.

Do you know off the top of your head of an example/pattern for making a single http request and getting the response from within a component method, as apposed to injecting it into the component with a 'connect' wrapper?

Reply:
oh, yup - you need to add an empty body param or the request reverts to using v1.1 for some reason.

Reply:
Doh! I get the use of requestHttpData now so no need to look for that 'example/pattern for making a single http request and getting the response from within a component method' @gwwar 

https://github.com/Automattic/wp-calypso/pull/36442/files#diff-33f4ec9b84efb2e6e29fcf410a637a40R25

Thanks @jeraldjuice for showing the proper use of this - I had mapping of state to props stuck in my head with this being being a data layer thing, but it definitely isn't a map state to props thing - https://github.com/Automattic/wp-calypso/pull/36442/files#r335114188